gcalcli/gcal.py

```diff
@@ -1464,32 +1491,56 @@ def ImportICS(self, verbose=False, dump=False, reminders=None,
                 continue
 
             self._add_reminders(event, reminders)
+
             if not verbose:
+                # Don't prompt, just assume user wants to import.
+                pass
+            else:
+                # Prompt for whether to import.
+                self.printer.msg('\n[S]kip [i]mport [q]uit: ', 'magenta')
+                val = input()
+                if not val or val.lower() == 's':
+                    continue
+                elif val.lower() == 'q':
+                    sys.exit(0)
+                elif val.lower() == 'i':
+                    # Import requested, continue
+                    pass
+                else:
+                    self.printer.err_msg('Error: invalid input\n')
+                    sys.exit(1)
+
+            # Import event
+            import_method = (
+                self.get_events().import_ if (
+                    self._event_should_use_new_import_api(event, self.cals[0]))
+                else self.get_events().insert)
+            try:
                 new_event = self._retry_with_backoff(
-                    self.get_events().insert(
-                        calendarId=self.cals[0]['id'], body=event
-                    )
+                    import_method(calendarId=self.cals[0]['id'], body=event))
+            except HttpError as e:
+                try:
+                    is_skipped_dupe = any(detail.get('reason') == 'duplicate'
+                                          for detail in e.error_details)
+                except Exception:
+                    # Fail gracefully so weird error responses don't blow up.
+                    is_skipped_dupe = False
+                event_label = (
+                    f'"{event["summary"]}"' if event.get('summary')
+                    else f"with start {event['start']}"
                 )
+                if is_skipped_dupe:
+                    # TODO: #492 - Offer to force import dupe anyway?
+                    self.printer.msg(
+                        f'Skipped duplicate event {event_label}.\n')
```

Comment:
FYI in my testing the API didn't seem to respect the "sequence" number and still considered the events duplicates.

And another confusing corner case: seems like deleted events can still be considered "duplicates" and skipped.

Those could both be helped if we added more explicit detection and prompted users for how to resolve weird cases.